source/install/self-managed-changelog.md

```diff
@@ -253,8 +253,16 @@ Mattermost v7.9.0 contains a low severity level security fix. [Upgrading](https:
 
 ## Release v7.8 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
 
+- **v7.8.8, released 2023-07-07**
+  - Mattermost v7.8.8 contains medium severity level security fixes. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) to this release is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
+  - Mattermost v7.8.8 contains no database or functional changes.
+  - Fixed an issue where posts were not fully sanitized for audit output when a link preview was included.
+  - Updated prepackaged Playbooks plugin version to 1.36.2.
+  - Fixed an issue where line breaks were introduced when pasting hyperlinks in the chat.
+  - New feature flag ``DataRetentionConcurrencyEnabled`` was added to enable/disable concurrency for data retention batch deletion. New config setting  ``DataRetentionSettings.TimeBetweenBatchesMilliseconds`` was added to control the sleep time between batch deletions.
```

Comment:
Thanks @amyblais! Should we add and describe the default value of newly added config setting?